go.mod

```diff
@@ -11,10 +11,9 @@ require (
 	github.com/google/go-cmp v0.6.0
 	github.com/operator-framework/api v0.25.0
 	github.com/operator-framework/catalogd v0.12.0
-	github.com/operator-framework/helm-operator-plugins v0.2.1
+	github.com/operator-framework/helm-operator-plugins v0.2.2-0.20240520180534-f463c36fedf9
 	github.com/operator-framework/operator-registry v1.43.1
-	github.com/operator-framework/rukpak v0.20.1-0.20240510194410-99faf1c1763f
-	github.com/spf13/cobra v1.8.0
+	github.com/operator-framework/rukpak v0.21.2-0.20240606055133-ac1802e9c1e8
```

Comment:
Should we cut a new release for this? Our latest release is v0.21.1 and https://github.com/operator-framework/rukpak/commit/ac1802e9c1e8 is not in it.

Reply:
As far as I see `main` currently points at https://github.com/operator-framework/rukpak/commit/ac1802e9c1e8. So it is as recent as it can be as of right now :)

Reply:
Ok, I did update that. ;)

Reply:
Just to clarify what I was getting at: it looks like we are currently pinning rukpak to a specific commit (probably because of `go get github.com/operator-framework/rukpak@latest`).

Can we switch back to using tags as with catalogd, operator-registry, etc?

We used to use tags (see 6060534838ef1450bbe26c71d671a4c942fd80f3, for example), but it looks like with helm integration we merged non-release dependency.

Reply:
We would need a tag; there is no tag at the tip of rukpak. I asked for one in the channel.

Reply:
There's now `v0.22.0`